scripts/plotting/global_latlon_map.py

```diff
@@ -442,6 +375,21 @@ def global_latlon_map(adfobj):
                                 elif (redo_plot) and plot_name.is_file():
                                     plot_name.unlink()
 
+                                #If requested, then calculate the monthly-weighted seasonal averages:
+                                if weight_season:
+                                    mseasons[s] = pf.seasonal_mean(mdata, season=s, is_climo=True)
+                                    oseasons[s] = pf.seasonal_mean(mdata, season=s, is_climo=True)
```

Comment:
I think we'll want to use `odata` here:

```suggestion
                                    oseasons[s] = pf.seasonal_mean(odata, season=s, is_climo=True)
```